LiveSplit.AutoSplitters.xml

```diff
@@ -8939,6 +8939,18 @@
         <Description>LiveSplit AutoSplitter for Crow Story</Description>
         <Website>https://github.com/LRFLEW/LiveSplit.CrowStory</Website>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Melatonin</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/LRFLEW/LiveSplit.Melatonin/main/Melatonin.asl</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/main/lib/UnityASL.bin</URL>
```

Comment:
```suggestion
            <URL>https://github.com/just-ero/asl-help/raw/main/lib/asl-help</URL>
```

Reply:
Whoops. That was a bad copy-paste from elsewhere in the file. I've fixed it now.